package.json

```diff
@@ -18,17 +19,30 @@
     "express-cache-controller": "^1.1.0",
     "limiter": "^1.1.5",
     "mariadb": "^2.4.1",
-    "memory-cache": "^0.2.0"
-    },
+    "memory-cache": "^0.2.0",
+    "module": "^1.2.5",
+    "node-fetch": "^2.6.0",
+    "regenerator-runtime": "^0.13.5",
+    "sql": "^0.78.0",
+    "typescript": "^3.9.6"
+  },
   "devDependencies": {
+    "@babel/core": "^7.10.4",
+    "@babel/node": "^7.10.4",
+    "@babel/preset-env": "^7.10.4",
     "eslint": "^7.3.1",
     "eslint-config-airbnb-base": "^14.2.0",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-prettier": "^3.1.4",
     "husky": "^4.2.5",
+    "jest": "^26.1.0",
+    "jest-puppeteer": "^4.4.0",
     "lint-staged": "^10.2.11",
-    "prettier": "^2.0.5"
+    "nodemon": "^2.0.4",
+    "prettier": "^2.0.5",
+    "puppeteer": "^5.1.0",
+    "supertest": "^4.0.2"
```

Comment:
guessing some of these are for the test that will be part of your next PR. can leave them for now but they are unrelated to what your are doing in this PR.